experimental/cc_slurm_nhc/readme.md

```diff
@@ -69,6 +69,12 @@ DRAINED node from deallocating (i.e enable keep alive)
 You can add your own health checks to the NHC framework. An example is azure_cuda_bandwidth.nhc, which is a CUDA bandwidthtest health check specifically for GPU's.
 You just add your custom health check to /etc/nhc/scripts and modify your nhc.conf file to use it (/etc/nhc/nhc.conf).
 
+## Run NHC via SLURM Epilog
+If you need to run NHC checks after a job completes (SLURM Epilog), then set NHC_EPILOG=1 in the configure_nhc.sh script.
+
+>Note: If you run NHC via Epilog, then set HealthCheckInterval to a large value so it effectively only runs when a new node is allocated.
```

Comment:
The term "allocated" used here in the CC context clashes with the Slurm terminology.
I would suggest using "provisioned in the cluster" or something along those lines.

Reply:
Thanks will make that change.